docs/diagrams/SearchCommandSequenceDiagram.puml

```diff
@@ -0,0 +1,83 @@
+@startuml
+/' Sequence Diagram for postal sector search '/
+!include style.puml
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":OrderBookParser" as OrderBookParser LOGIC_COLOR
+participant ":SearchCommandParser" as SearchCommandParser LOGIC_COLOR
+participant ":ArgumentTokenizer" as ArgumentTokenizer LOGIC_COLOR
+participant "argMultimap:ArgumentMultimap" as ArgumentMultimap LOGIC_COLOR
+participant "s:SearchCommand" as SearchCommand LOGIC_COLOR
+participant "order:OrderContainsKeywordsPredicate" as OrderContainsKeywordsPredicate LOGIC_COLOR
+participant "result:CommandResult" as CommandResult LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+end box
+
+[-> LogicManager : execute("search Alice")
+activate LogicManager
+
+LogicManager -> OrderBookParser : parseCommand("search Alice")
+activate OrderBookParser
+
+create SearchCommandParser
+OrderBookParser -> SearchCommandParser
+activate SearchCommandParser
+
+SearchCommandParser --> OrderBookParser
+deactivate SearchCommandParser
+
+OrderBookParser -> SearchCommandParser : parse("Alice")
+activate SearchCommandParser
+
+
+SearchCommandParser -> ArgumentTokenizer++ : tokenize(args, PREFIX_TID, ...)
```

Comment:
Because the working process is explained with details in the DeveloperGuide, the sequence diagram can mainly include the interaction between logic components.